en/docs/troubleshoot/faq.md

```diff
@@ -0,0 +1,147 @@
+
+## Billing and Support
+
+### Q: Whom do I reach out to if I have a billing question?  
+Please reach out to cloud-billing-support@wso2.com or create a support ticket via our support portal.
+
+### Q: What's a Developer plan?
+A Developer plan allows you to try out Choreo’s capabilities at no cost. It’s ideal for proof of concept (PoC)  tasks or workloads with limited transactions. This plan allows you to experiment with up to 5 components and provides $1,000/year of Choreo Data Plane (CDP) credits.
+
+### Q: How do I calculate the infrastructure costs?
+Calculating infrastructure costs depends on the type of workload you want to manage. Here are a few examples:
+
+- **Example 1**: Managing existing APIs with Choreo API Manager as an API proxy with simple mediation; no additional infrastructure costs.
+- **Example 2**: Managing existing APIs with Choreo API Manager as an API proxy with complex mediation and policies; Choreo will deploy 1 x container to handle these mediation and policies at approximately USD$57.25 per month per API.
+- **Example 3**: Creating, deploying, and managing a new API or integration within Choreo; pay for 1 x component + infrastructure cost. Each container deployed will be approximately USD$57.25 per month on the default configuration provided by Choreo. Additional resources will be charged based on the type of resource required.
+- **Example 4**: Creating, deploying, and managing a microservice; the same approach as example 3.
+
+### Q: What are the component limitations? 
+
+- **Developer plan**: Allows up to a maximum of five free components and unlimited paid components.
+- **PAYG plan**: Allows unlimited paid components.
+- **Enterprise plan**: Allows unlimited paid components.
+
+### Q: How do I read the bill?
+Your bill will detail the number of components used, infrastructure consumed, support plans used, and any additional services you may have purchased. If you are unsure about any charges on your bill, reach out to choreo-support@wso2.com for clarification. 
+
+### Q: Is support included in the Choreo Enterprise plan?   
+The Choreo Enterprise plan does not automatically include support; however, you can purchase support plans in addition to the Enterprise plan at any time. Find out more at https://wso2.com/choreo/customer-support/.
+
+### Q: I am an Enterprise subscription customer who wants to use the Choreo Private Data Plane. What costs will I incur in addition to the subscription and support plan?
+You can start by using a basic plan or contact us for an Enterprise Support plan.
+
+### Q: I want to upgrade from PAYG to an Enterprise subscription. Will there be an outage during the upgrade?
+No, there are no outages when upgrading a plan.
+
+## Choreo Internal Developer Platform
```

Comment:
Since we no longer have the profiles, are we going to have the profile-specific questions categorized in this manner?

Reply:
Yes, because we have the marketing pages for entry. Hence, users will need to know what they actually mean.